client-test/Makefile

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I don't think we need this by using docker gradle plugin and shadowJar.
I think pushing should be done separately from building.

Reply:
ok fixed in https://github.com/scalar-labs/kelpie-test/pull/2/commits/205579cd7434e36d825be604616d571d7c6d9bc5 and https://github.com/scalar-labs/kelpie-test/pull/2/commits/597c58a5e7d75aedbc45adcfdec8e21d86dcea8b